library/choose_disk.py

```diff
@@ -0,0 +1,436 @@
+#!/usr/bin/python
+
+from ansible.module_utils.basic import AnsibleModule
+import json
+import logging
+import os
+import re
+import subprocess
+
+DOCUMENTATION = '''
+---
+module: choose_disk
+author: Erwan Velu <erwan@redhat.com>
+short_description: Choose disks based on their features
+description:
+    To be completed
+'''
+
+logger = logging.getLogger('choose_disk')
+module = None
+
+
+def _equal(left, right):
+    return left == right
+
+
+def _gt(left, right):
+    return float(left) > float(right)
+
+
+def _lt(left, right):
+    return float(left) < float(right)
+
+
+_REGEXP = re.compile(r'^([^(]+)'          # function name
```

Comment:
this could be much simpler by relying only on the operands themselves: `>`, `<`. Chances are that more people will understand (and remember) a plain operand like `<` than using a custom name and syntax like `_lt()`. 

In the case of simple operands, there wouldn't be a need to parse with a regex, check if the operand exists in globals, and then execute the parsed value. It would be enough to check for existence of the operand and apply it directly. A simplistic example would look like: